### ISSUE TITLE
Tuple sorting doesn't consider profile

### ISSUE THREAD

**Issue by kasium** [PyCQA/isort#1815]
The tuple, list, etc sorting options don't consider the profile option.

Code:
```python
# isort: tuple
data = ("therearesuperlong", "therearesuperlong", "therearesuperlong", "therearesuperlong", "therearesuperlong")
```

`isort dummy.py --profile black`
```python
# isort: tuple
data = ('therearesuperlong', 'therearesuperlong', 'therearesuperlong', 'therearesuperlong',
 'therearesuperlong')
```

`black dummy.py`
```python
# isort: tuple
data = (
    "therearesuperlong",
    "therearesuperlong",
    "therearesuperlong",
    "therearesuperlong",
    "therearesuperlong",
)
```

I would expect that these sorting options consider the profile as normal. Also, the sorting options are not documented but noted  here #1358

**Comment by jherman3** [comment]
I'm also experiencing this with `# isort: list`. It'd be great to have this so we can sort `__all__`.

**Comment by tueda** [comment]
Actually, the tuple sorting should follow the `multi_line_output` option (or possibly a similar new option for the tuple sorting) rather than the profile.

As a first step, it would be OK if the output is compatible with black (as described in #1358), but in general [`literal.assignment`](https://github.com/PyCQA/isort/blob/be0fbd07817abcd5d2e62db9233b6633715d3405/isort/literal.py#L38) needs to implement calling a variant of [`wrap.import_statement`](https://github.com/PyCQA/isort/blob/99faae897f9da95cfd74ef9914c70313ed9abac3/isort/wrap.py#L10). (Maybe the author is aware of it, but he is too busy to implement it...)

**Comment by timothycrosley** [comment]
This is true for all literal / type sorts, and is why they aren't documented / advertised features, even though If they were in a good spot I would do so because I think they are useful! 

```As a first step, it would be OK if the output is compatible with black (as described in #1358), but in general literal.assignment needs to implement calling a variant of wrap.import_statement. (Maybe the author is aware of it, but he is too busy to implement it...)

```
That's exactly right, I've just been busy, will accept PRs for this or will get to it at some point

**Comment by DanielNoord** [comment]
Should be fixed by https://github.com/PyCQA/isort/pull/2576

**Comment by kasium** [comment]
Thanks 👍
